src/components/EventChat/CommentCard.tsx

```diff
@@ -184,10 +184,9 @@ const CommentCard = (props: EventCardProps) => {
             onChange={handleCommentMsgChange}
             multiline={true}
             rows="3"
+            placeholder={!commentMsg ? 'Add text...' : undefined}
```

Comment:
Placeholder text if the comment doesn't already have text. This only shows up if the card is in edit mode.